Source/read.f90

```diff
@@ -13404,6 +13404,7 @@ SUBROUTINE SET_CTRL_DEFAULTS
    ID            = 'null'
    LATCH         = .TRUE.
    INITIAL_STATE = .FALSE.
+   CONTROL_FORCE = .FALSE.
```

Comment:
@marcosvanella Sorry, this was my fault.

Reply:
Ok, thanks.